src/Features/CSharp/Portable/SignatureHelp/InvocationExpressionSignatureHelpProvider_MethodGroup.cs

```diff
@@ -56,7 +56,9 @@ internal partial class InvocationExpressionSignatureHelpProvider
             else if (invocationExpression.Expression is SimpleNameSyntax &&
                 invocationExpression.IsInStaticContext())
             {
-                methodGroup = methodGroup.Where(m => m.IsStatic);
+                // Local functions even inside static methods can have IsStatic == false by the compiler update for C# 8.0.
+                // Therefore, here is a workaround to support these functions in the signature help.
```

Comment:
Not sure I understand the comment here. Does "workaround" here imply this is a bug in the compiler API or just how it works and we should understand it? Similarly, does "C# 8.0" imply there's some langver behavior here...?

Reply:
Not really. A more accurate description would be "We always need to include local functions regardless of whether they are static."

Reply:
Updated the comment. Thank you, @333fred !